doc/nrf/ug_bt_mesh_node_removal.rst

```diff
@@ -0,0 +1,190 @@
+        bt_mesh_cdb_node_foreach(send_net_key_update, subnet);
+        return 0;
+    }
+
+    ...
+
+    uint8_t new_net_key[16] = { ... };
+    update_net_key(BT_MESH_NET_PRIMARY, new_net_key);
+
+Switching to the new key
+========================
+
+After uploading the new key, command the nodes to use it when sending messages. To do that, change the Key Refresh Phase to 2 by sending the Config Key Refresh Phase Set message with Transition field set to 0x2.
+
+Send the message to at least one node. The rest nodes will switch the Key Refresh Phase through Secure Network Beacon. But, in this case, it may take a while for all nodes to eventually switch the phase since the time between 2 consecutive Secure Network Beacons is ~10 seconds. You can speed this process up by sending the Config Key Refresh Phase Set message to all nodes that needs to be updated.
+
+After switching the Key Refresh Phase to 2, a node will decrypt messages using both, old and new keys, but encrypt only using the new key. Therefore, when updating a network key, make sure to update the phase first on the node running the Configuration Client model so that it is able to decrypt the Config Key Refresh Phase Status message from other nodes when they switch to the new network key.
+
+The snippet below demonstrates how to set the Key Refresh Phase on all nodes using |NCS| API:
+
+.. code-block:: c
+
+    static uint8_t send_key_refresh_phase_set(struct bt_mesh_cdb_node *node, void *user_data)
+    {
+        struct bt_mesh_cdb_subnet *subnet = user_data;
+        uint8_t phase;
+        uint8_t status;
+        int err;
+
+        err = bt_mesh_cfg_cli_krp_set(node->net_idx, node->addr, subnet->net_idx, subnet->kr_phase,
+                                      &status, &phase);
+        if (err || status) {
+            printk("Unable to set Key Refresh Phase on %#x (err %d, status %u)\n",
+                   node->addr, err, status);
+        } else {
+            printk("Set Key Refresh Phase to %u on %#x\n", phase, node->addr);
+        }
+
+        return BT_MESH_CDB_ITER_CONTINUE;
+    }
+
+    static int update_key_refresh_phase(uint8_t net_idx, uint8_t phase)
+    {
+        struct bt_mesh_cdb_subnet *subnet;
+
+        subnet = bt_mesh_cdb_subnet_get(net_idx);
+        if (!subnet) {
+            printk("Unable to get subnet\n");
+            return -EINVAL;
+        }
+
+        subnet->kr_phase = phase;
+        bt_mesh_cdb_subnet_store(subnet);
+
+        bt_mesh_cdb_node_foreach(send_key_refresh_phase_set, subnet);
+        return 0;
+    }
+
+    ...
+
+    update_key_refresh_phase(BT_MESH_NET_PRIMARY, BT_MESH_KR_PHASE_2);
+
+If the Key Refresh Phase to be changed through Secure Network Beacons, wait until all nodes have changed the Key Refresh Phase to 2. This can be done by sending the Config Key Refresh Phase Get message to a specific node. To retrieve the Key Refresh Phase from a node using |NCS| API, use :c:func:`bt_mesh_cfg_cli_krp_get`.
```

Comment:
```suggestion
If the Key Refresh Phase is suposed to be changed through Secure Network Beacons, wait until all nodes have changed the Key Refresh Phase to 2. This can be done by sending the Config Key Refresh Phase Get message to a specific node. To retrieve the Key Refresh Phase from a node using |NCS| API, use :c:func:`bt_mesh_cfg_cli_krp_get`.
```

Reply:
Chose Mia's suggestion with just `is`.